src/app/index-landing/index-landing.component.html

```diff
@@ -23,9 +31,9 @@
     <img class="h-48 p-4 ring-1 rounded-lg mx-auto" src="/assets/images/onboarding-images/de4ZAy.tif.png" />
 </div>
 <div class="w-9/12 mx-auto mt-8 space-y-6 text-patchwrq">
-    <div class="ring-4 bg-patchwrq ring-gray-900 truncate shadow-md  text-center py-3 rounded-full">
+    <button class="w-full ring-4 bg-patchwrq ring-gray-900 truncate shadow-md  text-center py-3 rounded-full">
```

Comment:
@theJayBecker can you make the buttons, into buttons?
the other two, so All three are